src/main/java/org/apache/sling/feature/cpconverter/vltpkg/VaultPackageAssembler.java

```diff
@@ -248,14 +245,6 @@ public void addDependency(@NotNull Dependency dependency) {
             IOUtils.copy(input, output);
         }
 
-        // copy the required resources
```

Comment:
TBH: These lines are totally unclear to me, package metadata is supposed to be located below `META-INF` never below `jcr_root`. I just removed them, because the original metadata are already contained in the transformed package and only `filter.xml` and `properties.xml` is replaced, which seems fine to me.

Reply:
Maybe we should leave this change for another issue? It seems to be unrelated

Reply:
I can do that, but I am pretty sure the metadata should not be put below `jcr_root`, but tracking separately is probably better.

Reply:
Done in https://github.com/apache/sling-org-apache-sling-feature-cpconverter/pull/62/commits/133f25c2c418f399ad6005cf2782239c762bc2ba